cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/common/MeasureEvaluator.java

```diff
@@ -410,6 +412,11 @@ protected void evaluateProportion(
             int populationSize,
             MeasureReportType reportType,
             EvaluationResult evaluationResult) {
+        // check populations
+        R4MeasureScoringTypePopulations.validateScoringTypePopulations(
+                groupDef.populations().stream().map(PopulationDef::type).collect(Collectors.toList()),
+                MeasureScoring.fromCode(groupDef.measureScoring().toCode()));
```

Comment:
Why do this?  Why not just pass groupDef.measureScoring() directly into the method?